website/views.py

```diff
@@ -36,6 +36,7 @@
 
 logger = logging.getLogger(__name__)
 preprints_dir = os.path.abspath(os.path.join(os.getcwd(), EXTERNAL_EMBER_APPS['preprints']['path']))
+registries_dir = os.path.abspath(os.path.join(os.getcwd(), EXTERNAL_EMBER_APPS['registries']['path']))
```

Comment:
Remove - unnecessary because registries is wrapped into ember_osf_web

Reply:
👌